assets/js/logic.js

```diff
@@ -1 +1,143 @@
-// empty for init
+/* eslint-disable max-classes-per-file */
+
+// Main book class
```

Comment:
- [ ] OPTIONAL
Kindly ensure to remove **all** comments before pushing code to the repo to keep your code cleaner.